docusaurus/docs/create-a-plugin/develop-a-plugin/set-up-github-workflows.md

```diff
@@ -22,11 +22,15 @@ The CI (`ci.yml`) workflow is designed to lint, type check, and build the fronte
 
 ## The release workflow
 
+The release (`release.yml`) workflow is designed to build, test, package and sign your plugin whenever you're ready to release a new version. This automates the process of creating releases in GitHub and provides instructions for submitting the plugin to the Grafana plugin catalog.
+
 :::warning
 
 This workflow requires a Grafana Cloud API key. Before you begin, follow the instructions for [generating an Access Policy token](../../publish-a-plugin/sign-a-plugin#generate-an-access-policy-token).
 
-You can store your Access Policy token as a Repository Secret in GitHub:
+:::
+
+### Storing your Access Policy token as a Repository Secret in GitHub
```

Comment:
In light of the [Grafana style guide](https://grafana.com/docs/writers-toolkit/write/style-guide/capitalization-punctuation/), terms are lower case unless they are proper names or product names, and sentence case if they are UI elements. This leaves terms like "Access Policy token" and "Repository Secret" a bit ambiguous. I would argue that the former is a proper name for CLI element, similar to UI elements, and the latter is a common noun. Accordingly, I would change "Repository Secret" to "repository secret".